test/cpp/qps/qps_driver.cc

```diff
@@ -68,49 +82,124 @@ using grpc::testing::ServerType;
 using grpc::testing::RpcType;
 using grpc::testing::ResourceUsage;
 
+using std::vector;
+using std::string;
+
+vector<string> ParseCommaSeparatedIntList(string list) {
+  // Pass by value because strtok consumes its input.
+  vector<string> result;
+  char *p = strtok((char*)list.c_str(), ",");
```

Comment:
There must be some alternative out there to strtok, right? Yes, this function is not being used in a multithreaded context, but it is conceivable that somebody will want to copy the code for some other purpose and get burned in the process.